Cargo.toml

```diff
@@ -10,3 +10,5 @@ members = [
 [patch.crates-io]
 prost-derive = { git = "https://github.com/danburkert/prost" }
 
+[patch."https://github.com/tower-rs/tower-h2"]
+tower-h2 = { git = "https://github.com/hawkw/tower-h2", branch = "eliza/new-tokio" }
```

Comment:
Is there a PR related to this?

Reply:
tower-rs/tower-h2#27

I was originally planning to just use the branch in the tower-h2 repository, but in order to patch a dependency, it needs to come from a different source. Thus the fork.

Reply:
It looks like that PR only impacts examples? If it's not strictly required, it would be good to separate. Otherwise, we should resolve that PR before this goes through.

Reply:
I _thought_ that the examples failing to compile broke my build of Conduit, but I just tried it again with the master branch and it worked. Will remove this change.

Reply:
Hmm, interesting --- it looks like tower-rs/tower-h2@d9b3140e543af7887893cbf2f4337afd28d15839 broke several of our metrics tests, I have yet to figure out exactly why (it appears that stats which update on request ends are no longer updated). 

So we can build without my `tower-h2` branch, but the latest commit breaks the build.